tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -636,21 +638,28 @@ def args_dict_to_list(cli_parser, **kwargs):
 
 def get_non_default_params(argv, cli_parser):
     import numbers
+    import inspect
+    from openvino.tools.mo import convert_model
+
+    signature = inspect.signature(convert_model)
     # make dictionary with parameters which have non-default values to be serialized in IR in rt_info
     non_default_params = {}
     for arg, arg_value in vars(argv).items():
-        if arg_value != cli_parser.get_default(arg):
-            value = depersonalize(arg_value, arg)
-            # Skip complex classes in params to prevent
-            # serializing it to rt_info
-            if isinstance(value, (str, bool, numbers.Number)):
-                non_default_params[arg] = value
+        if arg in signature.parameters and arg_value == signature.parameters[arg].default:
+            continue
+        if arg_value == cli_parser.get_default(arg):
```

Comment:
can it be a case that default values in cli_parser and convert_model are different and the param value turns to be not default?

Reply:
Currently all default values for cli tool and for convert_model() are the same.

In this line I check default value from both signature and from cli parser because some parameters present only in cli parser (output_dir, model_name) or only in convert_model (PyTorch params). So default values for them are stored in signature for convert_model and in cli parser for cli tool.

If we want to make this logic work for different default values if we want to have it in future, then we need to check here which path was used (convert_model or cli tool) and check corresponding location of default params. I can add this logic if necessary.